Makefile

```diff
@@ -65,6 +65,13 @@ delete-kind-cluster:
 fmt: ## Run go fmt against code.
 	go fmt ./...
 
+.PHONY: fmt-njs
+fmt-njs: ## Run prettier against the njs httpmatches module.
+	docker run --rm \
```

Comment:
Would it make sense to add  issue/task to integrate Javascript with the GitHub pipeline (linting, formatting) ? https://github.com/nginxinc/nginx-kubernetes-gateway/tree/main/.github/workflows

Also, unit tests are run the container, but the it might be possible to use a pre-built action to run the tests in the pipeline.

Reply:
Yes, definitely. I'll add one

Reply:
Added

Reply:
@pleshakov I went ahead and added lint and test jobs for the njs code to the workflow to this PR: https://github.com/nginxinc/nginx-kubernetes-gateway/pull/109/commits/f8cccb9e1fd77bdf76a488f6380104fb932ef752

I'm happy to open a separate PR for this if you'd prefer.